pom.xml

```diff
@@ -104,13 +105,16 @@
     <numForkedIT>7</numForkedIT>
     <it.failIfNoSpecifiedTests>false</it.failIfNoSpecifiedTests>
     <surefire.failIfNoSpecifiedTests>false</surefire.failIfNoSpecifiedTests>
+    <skip-phoenix5-assembly>false</skip-phoenix5-assembly>
+    <skip-phoenix4-assembly>false</skip-phoenix4-assembly>
```

Comment:
Come to think of it, it would save build/test time if we could also skip building the phoenix4/5 modules alltogether.

Reply:
That is a great idea!